src/main/java/org/alliancegenome/curation_api/services/base/CurieObjectCrudService.java

```diff
@@ -0,0 +1,44 @@
+package org.alliancegenome.curation_api.services.base;
+
+import org.alliancegenome.curation_api.dao.base.BaseEntityDAO;
+import org.alliancegenome.curation_api.model.entities.base.CurieObject;
+import org.alliancegenome.curation_api.response.ObjectResponse;
+import org.alliancegenome.curation_api.response.SearchResponse;
+
+import io.quarkus.logging.Log;
+import jakarta.transaction.Transactional;
+
+public abstract class CurieObjectCrudService<E extends CurieObject, D extends BaseEntityDAO<E>> extends BaseEntityCrudService<E, D> {
+
+	public ObjectResponse<E> get(String curie) {
+		E object = findByCurie(curie);
+		ObjectResponse<E> ret = new ObjectResponse<E>(object);
+		return ret;
+	}
+
+	@Transactional
+	public ObjectResponse<E> delete(String curie) {
+		E object = findByCurie(curie);
+		if (object != null)
+			dao.remove(object.getId());
+		ObjectResponse<E> ret = new ObjectResponse<>(object);
+		return ret;
+	}
+	
+	public E findByCurie(String curie) {
```

Comment:
I thought this kind of logic was better kept in the service classes rather than DAO.  Also, I only wanted it accessible by the classes that inherit from CurieObject